From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 28 Apr 2021 14:27:30 +0100
Subject: spelling

---
 src/App/ElnAssert.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/App/ElnAssert.cpp b/src/App/ElnAssert.cpp
index 1cc193d..7afe4c6 100644
--- a/src/App/ElnAssert.cpp
+++ b/src/App/ElnAssert.cpp
@@ -173,7 +173,7 @@ void Assertion::reportFailedToSave(QString msg2) {
     msg += ".";
 
   msg += "\nWhile trying to save your most recent changes,"
-    " another problem occured:";
+    " another problem occurred:";
   msg += "\n" + msg2.trimmed();
   if (!msg.endsWith("."))
     msg += ".";
